src/runtime/internal/utils.ts

```diff
@@ -189,3 +189,15 @@ export const has_prop = (obj, prop) => Object.prototype.hasOwnProperty.call(obj,
 export function action_destroyer(action_result) {
 	return action_result && is_function(action_result.destroy) ? action_result.destroy : noop;
 }
+
+export function split_css_unit(value: number | string, fallback = 'px'): [number, string] {
+	if (typeof value === 'number') {
+		return [value, fallback];
+	}
+	const split = value?.match?.(/^\s*(-?[\d.]+)([^\s]*)\s*$/);
+	if (split) {
+		return [parseFloat(split[1]), split[2] || fallback];
+	}
+	console.warn('Failed to split', value);
+	return [parseFloat(value), fallback];
+}
```

Comment:
I vote for shaving of as many bytes as possible, therefore removing the warning and removing the fallback parameter because that isn't used in production anywhere.
```suggestion
export function split_css_unit(value: number | string): [number, string] {
	if (typeof value === 'number') {
		return [value, 'px'];
	}
	const split = value?.match?.(/^\s*(-?[\d.]+)([^\s]*)\s*$/);
	if (split) {
		return [parseFloat(split[1]), split[2] || 'px'];
	}
	return [parseFloat(value), 'px'];
}
```